hack/init-environment.sh

```diff
@@ -231,6 +231,8 @@ function install_cloudsql_proxy () {
   local -r region=$(vault read -field=region $vault_location)
   local -r project=$(vault read -field=project $vault_location)
 
+  local -r vault_sa_location=secret/dsde/monster/${env}/command-conter/gcs/sa-key
```

Comment:
Found a small typo: `command-conter` should be `command-center`

Reply:
Also I think you might need to explicitly read in the field to get the `sa-key`. Something like this perhaps: 
```
local -r vault_sa_location=secret/dsde/monster/dev/command-center/gcs
local -r name=$(vault read -field=sa-key $vault_sa_location)
```

Reply:
Good catch on the typo lol. How come we weren't explicitly reading the key before though?

Reply:
Otherwise, the changes to this file look good to me 😁